libafl_bolts/src/shmem.rs

```diff
@@ -247,6 +247,15 @@ pub trait ShMem: Sized + Debug + Clone + DerefMut<Target = [u8]> {
         }
     }
 
+    /// Consume current shared memory structure, and get the raw pointer to
+    /// this shared memory.
+    ///
+    /// Note that calling this method will result in a memory leak.
+    fn into_raw<T: Sized>(self) -> *mut T {
```

Comment:
I don't 100% get the need for this function, why can't we keep a reference to the shmem around in general?

Reply:
The shms are globally owned and we will need lots of global OnceCell to keep them. Note we can’t change EDGES_MAP or so to OnceCell<SHM> because of symbol names. Therefore, I guess the intention is to avoid such duplicate cells? I’m okay with either way.

Reply:
Yes, just as @wtdcode said, the whole forksever logic relies on those global raw pointers, if we maintain those raw pointers and the shmem at the same time, we are creating potentially two mutable references to the same memory, which I think the memory leak is a better choice. And the original version of forksever also leaks the memory.

Reply:
OK.